Cargo.toml

```diff
@@ -15,4 +15,4 @@ features = ["std"]
 
 [profile.opt]
 inherits = "release"
-debug = 1
+debug = 1
```

Comment:
Trailing newline seems to be removed 👀 not very critical, I'll just merge this as is.